dbt/models/reporting/schema.yml

```diff
@@ -1,32 +1,157 @@
+                  AND triad IS NOT NULL
+                  AND geography_type IS NOT NULL
+                  AND property_group IS NOT NULL
+                  AND assessment_stage IS NOT NULL
+                  AND sale_year IS NOT NULL
+          - expression_is_true:
+              name: ratio_stats_within_directionality
+              expression: >-
+                within_20_pct >= within_10_pct
+                  AND within_10_pct >= within_05_pct
 
 models:
-  - name: reporting.res_report_summary
-    description: '{{ doc("table_res_report_summary") }}'
-    config:
-      tags:
-        - daily
-
+  - name: res_report_summary
+    tests:
+    - unique_combination_of_columns:
+        name: res_report_summary_unique_by_geography_id_property_group_assessment_stage_year
+        combination_of_columns:
+          - geography_id
+          - property_group
+          - assessment_stage
+          - year
+    - expression_is_true:
+        name: res_report_summary_sale_year_equals_year_minus_one
+        expression: CAST(sale_year AS INTEGER) = CAST(year AS INTEGER) - 1
+        select_columns:
+          - sale_year
+          - year
+    - expression_is_true:
+        name: res_report_summary_no_nulls
+        expression: >-
+          triad IS NOT NULL
+            AND geography_type IS NOT NULL
+            AND property_group IS NOT NULL
+            AND assessment_stage IS NOT NULL
+        select_columns:
+          - triad
+          - geography_type
+          - property_group
+          - assessment_stage
   - name: reporting.vw_assessment_roll
-    description: '{{ doc("view_vw_assessment_roll") }}'
-
+    tests:
+      - unique_combination_of_columns:
+          name: vw_assessment_roll_unique_by_year_stage_township_name_class
+          combination_of_columns:
+            - year
+            - stage
+            - township_name
+            - class
+      - expression_is_true:
+          name: vw_assessment_roll_no_nulls
+          expression: >-
+            stage IS NOT NULL
+              AND township_name IS NOT NULL
+              AND triad IS NOT NULL
+              AND class IS NOT NULL
   - name: reporting.vw_pin_most_recent_boundary
-    description: '{{ doc("view_vw_pin_most_recent_boundary") }}'
-
+    tests:
+      - unique_combination_of_columns:
+          name: vw_pin_most_recent_boundary_unique_by_pin10_municipality_name
+          combination_of_columns:
+            - pin10
+            - municipality_name
   - name: reporting.vw_pin_most_recent_sale
-    description: '{{ doc("view_vw_pin_most_recent_sale") }}'
-
+    tests:
+      - unique_combination_of_columns:
+          name: vw_pin_most_recent_sale_unique_by_pin
+          combination_of_columns:
+            - pin
+      - value_is_present:
+          name: vw_pin_most_recent_sale_null
+          expression: sale_price IS NULL
+      - value_is_present:
+          name: vw_pin_most_recent_sale_not_null
+          expression: sale_price IS NOT NULL
```

Comment:
**issue:** Can you add some context about what this is testing in a YAML comment?